Cargo.toml

```
@@ -1,33 +1,36 @@
 [package]
 name = "zip-extract"
-version = "0.1.3"
+version = "0.2.0"
```

```suggestion
version = "0.1.3"
```

I agree that it should be released as 0.2.0, but I'll let cargo-release do it after the merge.